src/strings.js

```diff
@@ -1,25 +1,27 @@
-function sayHello (string) {
-  // your code here
+function sayHello(string) {
+  const hello = `Hello, ${string}!`;
+  return hello;
```

Comment:
instead of assigning the constant to just return it, just return the value, i.e. 
```
return `Hello, ${string}!`;
```